protovalidate/internal/extra_func.py

```diff
@@ -23,6 +23,12 @@
 from protovalidate.internal import string_format
 from protovalidate.internal.rules import MessageType, field_to_cel
 
+_USE_RE2 = True
+try:
+    import re2  # type: ignore
```

Comment:
one other minor thing, if we want: could install the [`google-re2-stubs` package](https://pypi.org/project/google-re2-stubs/) as a dev dependency, which would give us type info for re2 (and we could nix the ignore here).

ref: https://github.com/ddn0/google-re2-stubs